File: programming/labaratory-work-1/solutions/solution-2/go/main.go
package main

import "fmt"

func main() {
	var n int
	var m int
	var sum int = 0

	fmt.Scanf("%d", &n)
	fmt.Scanf("%d", &m)

	for i := 0; i < m; i++ {
		sum += n % 10
		n /= 10
	}

	fmt.Printf("%d", sum)
}

File: programming/labaratory-work-1/solutions/solution-2/main.c
#include <stdio.h>

int main() {
  int n = 0;
  int m = 0;
  int sum = 0;

  scanf_s("%d", &n);
  scanf_s("%d", &m);

  for (int i = 0; i < m; i++) {
    sum += n % 10;
    n /= 10;
  }

  printf_s("%d", sum);

  return 0;
}